enhancements/microshift/kubernetes-for-device-edge.md

```diff
@@ -0,0 +1,508 @@
+---
+title: kubernetes-for-device-edge
+authors:
+  - dhellmann
+  - fzdarsky
+  - derekwaynecarr
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@deads2k, control plane expert"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@jogeo, MicroShift contributor"
+  - "@mangelajo, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@sallyom, MicroShift contributor"
```

Comment:
@mfojtik @tkashem @stlaz @ingvagabund @hasbro17 Please let me know which of your team members to include as reviewers here.